lib/cc/analyzer/source_fingerprint.rb

```diff
@@ -28,7 +28,12 @@ def relevant_source
       end
 
       def raw_source
-        @raw_source ||= File.read(issue.path)
+        @raw_source ||=
+          if File.exist?(issue.path)
```

Comment:
Thoughts on `File.file?` which is "exists and is not a directory". The exceptions that prompted some other directory-related work were from such a case (but didn't fix it because of this unknown order situation).